### DISCUSSION TITLE
Can not start server in the example

### DISCUSSION THREAD

**Discussion by icespacelivecn** [zino-app/graphql-flutter#1165] [General]
Checked the dependency in the example server.
dependency_overrides:
  graphql_server:
    git:
      url: git@github.com:micimize/angel.git
      ref: typeerror2.8-fix
      path: packages/graphql/graphql_server
  graphql_parser:
    git:
      url: git@github.com:micimize/angel.git
      path: packages/graphql/graphql_parser

Found that the graphql_server and graphql_parser gits no longer exist.

**Comment by vincenzopalazzo** [comment]
Mh! I did not test it, but we are planning to remove the examples and make some good ones with a maintained server.

Do you need the server, for something in particular?

**Comment by vincenzopalazzo** [comment]
Ok the server is buggy and I'm working on it to provide a open source solution that did not disappear, and also I would like to provide a way where the user can use the open API to make a reproducible example before open a issue.

Sorry for the inconvinient
